content/rest-api/permissions.md

```diff
@@ -47,8 +47,11 @@ Do not give this permission to your UI user role as it makes no sense to give AP
 :::
 
 :::info
-Note that if a role has `Overall Web API` access, then it means that all the users under that role will be able to make requests on products, product models and published products.  
-There is no way to only restrict access to products, except if you are using a 2.x Enterprise Edition. In this case, the EE permissions based on user groups are applied to the API for the products and the published products.
+**Until the 5.0 version**, users with `Overall Web API` were able to make requests on products and product models without additional permissions.
+
+On SaaS and newer versions, new ACLs were introduced: `Create and update products`, `List products` and `Delete products`.
```

Comment:
```suggestion
We introduced the following ACLs on **Akeneo SaaS versions** for more security: `Create and update products`, `List products` and `Delete products`.
```

Reply:
I have a proofreading timeslot on Thursday with Camille. I'll check with her if that's ok to change this sentence. I'd like to add the Security argument to explain why we made it.